os/pm/pm_resume.c

```diff
@@ -109,8 +110,24 @@ int pm_resume(enum pm_domain_e domain)
 		goto errout;
 	}
 	g_pmglobals.suspend_count[domain]--;
+	/* If LCD resume the pm operation then start the timer */
+	if ((domain == PM_LCD_DOMAIN) && (g_pmglobals.state < PM_STANDBY) && (g_pmglobals.suspend_count[PM_LCD_DOMAIN] == 0)) {
```

Comment:
How about move this code into pm_timer_resume. and call it only here?
And renam pm_timer_resume to pm_process_resume?

Reply:
Done